tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -90,7 +102,7 @@
           <span>[see Section 2 below for details].</span>
         </p>
         <p id="summary-host">
-          <span>[[_host_conclusion]]</span>
+          <b>Recommendation for next step: <span class="host-conclusion">[[_host_conclusion]]</span></b>
```

Comment:
Instead of using the `<b>` tag here, remove it, and bold this paragraph in the CSS. Like this:
```css
#summary-host {
  font-weight: bold;
}
```
By only using CSS to style, we separate style from content.

Reply:
Done